Demo/src/main/java/com/paypal/android/ui/card/CardViewModel.kt

```diff
@@ -77,10 +78,13 @@ class CardViewModel @Inject constructor(
         val card = Card(number, monthString, yearString)
         card.securityCode = _securityCode.value ?: ""
 
+        System.setProperty("magnes.debug.mode", true.toString()) // check this
```

Comment:
Should this line or the comment be removed?

Reply:
this is the only way to make it log. Perhaps we could make `PayPalDataCollector` have a method like `setLogging(boolean)`. The only thing is that in the logs they will see the `Magnes` tag, do we want the merchants to know that we use magnes?

Reply:
I guess I was mostly wondering about the //check this comment. Are we telling ourselves or the merchant to check this?

Reply:
yeah my bad, that was more for us. I added the function `setLogging`, to give merchants a chance to see if data collection fails.